hack/make-rules/clean.sh

```diff
@@ -24,15 +24,11 @@ source "${KUBE_ROOT}/hack/lib/util.sh"
 CLEAN_PATTERNS=(
   "_tmp"
   "doc_tmp"
-  "((?!staging\/src\/k8s\.io\/apiextensions-apiserver\/pkg\/generated\/openapi).)*/zz_generated.openapi.go"
```

Comment:
but  ... this things are no longer cleaned ??

Reply:
This regex matches nothing and all openapi files are properly commeiited now.